Sources/ContentfulPersistence/SynchronizationManager.swift

```diff
@@ -123,6 +128,68 @@ public class SynchronizationManager: PersistenceIntegration {
         }
         self.localeCodes = localeCodes
     }
+    
+    /// Synchronizes data using the provided localization scheme, performing a follow-up sync with all localizations if required.
+    /// - Parameters:
+    ///   - syncSpacePersistable: The type conforming to `SyncSpacePersistable` used for handling synchronization data.
+    ///   - initialLocalizationScheme: The localization scheme to use for the initial synchronization phase.
+    ///     This must not be `.all`.
+    ///   - limit: An optional limit for the number of records to synchronize in one batch. Defaults to `nil`, indicating no limit.
+    ///   - dbVersion: The database version targeted for synchronization. Defaults to the application's default database version.
+    ///   - onInitialCompletion: A closure called upon completion of the initial synchronization phase, with a `Result` containing either the `SyncSpace` or an error.
+    ///   - onFinalCompletion: A closure called upon completion of the final synchronization phase, with a `Result` containing either the `SyncSpace` or an error.
+    /// - Throws: An assertion failure if `initialLocalizationScheme` is `.all`.
+    ///
+    /// - The synchronization process involves two phases:
+    ///     1. Synchronizing data with the provided localization scheme.
+    ///     2. Resetting the synchronization token and synchronizing with `.all` localizations.
+    ///     The `onInitialCompletion` closure is invoked after the first phase, and the final result is passed to the `onFinalCompletion` closure.
+    public func sync<SSP>(
```

Comment:
SSP does not say much maybe could be slightly longer self-explaining variable name?

Reply:
It's not a variable, but the generic placeholder, which by default tends to be `T`.
I just used `SSP` as the acronym from the protocol: `SyncSpacePersistable`. Do you have any suggestions? 🙏

Reply:
Oh yeah, that threw me off :D Maybe name it T? :D